go.mod

```diff
@@ -53,7 +53,7 @@ require (
 	github.com/grpc-ecosystem/grpc-gateway v1.5.1 // indirect
 	github.com/hashicorp/consul v0.0.0-20171026175957-610f3c86a089
 	github.com/hashicorp/errwrap v1.0.0
-	github.com/hashicorp/go-azure-helpers v0.0.0-20181126135526-ec113df69f49
+	github.com/hashicorp/go-azure-helpers v0.0.0-20181211121309-38db96513363
```

Comment:
I may be misunderstanding go modules but why can't we pin this to a tag, like many other dependencies here? i.e. why can't this line be
```
github.com/hashicorp/go-azure-helpers v0.3.1
```
?